CMakeLists.txt

```diff
@@ -883,8 +883,15 @@ if(EMSCRIPTEN)
     )
 
     add_executable( ktx_js interface/js_binding/ktx_wrapper.cpp )
```

Comment:
Please add a KTX_FEATURE_JS_WRITE option only visible `if(EMSCRIPTEN)` and defaulting to OFF. The documentation string should say something like "JS wrapper with minimal write functions as used by glTF-Compressor.". Only add ktx_js when set to ON.

Reply:
Discussion has been helpful. For example I was unaware the cmake_dependent_option was introduced in 3.22. Our CMakeLists.txt uses it a lot. I've just created PR #904 to bump the minimum version to 3.22 after checking that all our environments have at least that.

But yes. I'll take over this PR. On reflection I don't think `KTX_FEATURE_JS_WRITE` is needed.  We just need to comment out the addition of the read-write version to the release archive so we don't release it until such time as has fully fledged write features.